addons/binding/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/internal/discovery/DSMRDiscoveryService.java

```diff
@@ -67,10 +67,9 @@ public DSMRDiscoveryService() {
     public boolean meterDiscovered(DSMRMeterDescriptor meterDescriptor, ThingUID dsmrBridgeUID) {
         DSMRMeterType meterType = meterDescriptor.getMeterType();
         ThingTypeUID thingTypeUID = meterType.getThingTypeUID();
-        String thingId = "dsmr:" + meterType.name().toLowerCase() + ":"
-                + (meterDescriptor.getChannel() == DSMRMeterConstants.UNKNOWN_CHANNEL ? "default"
-                        : meterDescriptor.getChannel());
-        ThingUID thingUID = new ThingUID(thingId);
+        String thingId = ":" + (meterDescriptor.getChannel() == DSMRMeterConstants.UNKNOWN_CHANNEL ? "default"
```

Comment:
Thanks for the update! Though I think this ":" here should also be removed.

Reply:
oeps.. yes the `:` is the default separator.